charts/vector/Chart.yaml

```diff
@@ -18,11 +18,11 @@ maintainers:
   - name: Datadog
```

Comment:
`version` above needs bumping!

Reply:
I noticed that, but chose not to do it manually because the script didn't, and in this patch PR, it was also not edited:

https://github.com/vectordotdev/helm-charts/pull/293/files

🤔

Reply:
Is that a bug in the script ?

Reply:
Ah, we could probably automate it. You got caught here because this is the first PR after the previous release. The linked PR didn't have to change it as it had already been bumped so it was "ready"

Reply:
Ohh. I suppose the README version needs to change as well then

Reply:
Yeah, I swear that's mentioned in the docs somewhere... maybe there's release docs in confluence? Or maybe I'm thinking of OPW.